packages/data-portal-explore/src/components/FileTable.tsx

```diff
@@ -571,6 +571,17 @@ export class FileTable extends React.Component<IFileTableProps> {
                 wrap: true,
                 sortable: true,
             },
+            {
+                name: 'Treatment',
+                selector: (file: Entity) => {
+                    return _.uniq(
+                        file.diagnosis.map((d) => d.TreatmentType)
```

Comment:
we should read the value from `therapy` instead of `diagnosis`

```suggestion
                        file.therapy.map((d) => d.TreatmentType)
```

Reply:
Thank you for pointing it out, that was an oversight from my end.